relay-server/server/relayServer.go

```diff
@@ -844,3 +858,139 @@ func (rs *RelayServer) GetFeedsFromNodes() {
 		}
 	}
 }
+
+func startIPInformers(ctx context.Context, wg *sync.WaitGroup) {
+	kg.Printf("Started IP informers")
+	defer wg.Done()
+
+	// Check if context is already canceled
+	if ctx.Err() != nil {
+		kg.Errf("Context canceled before starting informers")
+		return
+	}
+
+	factory := informers.NewSharedInformerFactory(K8s.K8sClient, 30*time.Second)
+	podInformer := factory.Core().V1().Pods().Informer()
+	svcInformer := factory.Core().V1().Services().Informer()
+
+	updateIP := func(kind, namespace, name, ip string) {
+
+		if ip == "" {
+			Ipcache.Delete(ip)
+			return
+		}
+		resource := ""
+		switch strings.ToUpper(kind) {
+
+		case "POD":
+			resource = fmt.Sprintf("%s.%s.pod", name, namespace)
```

Comment:
Can we do k8s style formatting.
```suggestion
			resource = fmt.Sprintf("pod/%s/%s", namespace, name)


```